library/python/bytes_view.cc

```diff
@@ -28,7 +28,6 @@ envoy_data pyBytesAsEnvoyData(py::bytes bytes) {
 // we just copy it over to python-land.
 py::bytes envoyDataAsPyBytes(envoy_data data) {
   auto bytes = py::bytes((const char*)data.bytes, data.length);
-  release_envoy_data(data);
```

Comment:
again, that's kind of a convention that if your method receives envoy raw struct it is responsible for releasing it - @goaway told me about this convention. In his words:


>the convention is always as follows:
>1. if you are passed or returned a bridge struct you must assume the passer/returner is >relying on you to call release
>2. if you pass or return a bridge struct you assume the passee/returnee will cause release to >be called on it at some point exactly once
>
>in most cases that means if you receive a bridge struct you should call release on it when >you’re done. however, an exception is if you receive it and then pass it to something else (e.g. in delegation). in line with the two rules above, now you should assume whoever you >passed it to will be calling release at some point, thus absolving you of that responsibility.
>one more special case deserves mention - when you want to pass or return a bridge struct, >but continue to use it after doing so. in this case you have two options:
>1. pass a deep copy.
>2. pass a shallow copy with release replaced with a no-op, so you don’t have to worry about >when the passee/returnee calls it.